algorithms/symmetry/cosym/__init__.py

```diff
@@ -175,10 +175,12 @@ def _map_space_group_to_input_cell(intensities, space_group):
                 sg_best = sg_primitive.change_basis(cb_op_best_primitive.inverse())
                 # best_subgroup above is the bravais type, so create thin copy here with the
                 # user-input space group instead
+                best_subsym = best_subsym.customized_copy(
```

Comment:
I think this stuff could do with some kind of annotation somewhere... I am certain you have gained some understanding as a side-effect of these changes which should probably be documented inline?i.e. the _why_ of `est_subsym.change_basis(cb_op_inp_best.inverse())` etc.

I ask this as someone who may have to debug this one day :-)

Reply:
This is just to obtain the `subsym` in the input setting rather than the "best" setting -> `cb_op_inp_best.inverse()`